src/m_cheat.c

```diff
@@ -1046,6 +1068,53 @@ boolean M_FindCheats(int key)
   return ret;
 }
 
+typedef struct {
+  int input;
+  const cheat_when_t when;
+  const cheatf_t func;
```

Comment:
`cheatf_t` is a union. I think you will have to initialize it using the explicit element name, e.g. `.v = cheat_god` or `.i = cheat_pw`.

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/1001/commits/05fb48983d7ae16ba1b41557b237e40a497d28c3

Reply:
Even better! 👍